src/Components/Votes.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+
+export const Votes = () => {
+  const [votes, setVotes] = useState(0);
+  const [isClicked, setIsClicked] = useState(false);
+
```

Comment:
You've done great handling this voting in state - however adding the votes needs to persist in the database, it needs to be done alongside a call to the API to increase the votes on the article - optimistically 🙂 
You can find an example of this in the [notes](https://notes.northcoders.com/courses/js-front-end/ui-behaviours-and-optimistic-rendering)
You'll know if you've got this working because the latest votes will persist if you refresh the page